extension/chrome/elements/compose-modules/formatters/signed-msg-mail-formatter.ts

```diff
@@ -34,12 +34,14 @@ export class SignedMsgMailFormatter extends BaseMailFormatter {
       const allContacts = [...newMsg.recipients.to || [], ...newMsg.recipients.cc || [], ...newMsg.recipients.bcc || []];
       ContactStore.update(undefined, allContacts, { last_use: Date.now() }).catch(Catch.reportErr);
       const body = { 'text/plain': signedData };
-      return await SendableMsg.create(this.acctEmail, { ...this.headers(newMsg), body, atts });
+      //return await SendableMsg.create(this.acctEmail, { ...this.headers(newMsg), body, atts });
+      return await SendableMsg.createOpenPGPNoDraft(this.acctEmail, { ...this.headers(newMsg), body, atts });
     }
     // pgp/mime detached signature - it must be signed later, while being mime-encoded
     // prepare a sign function first, which will be used by Mime.encodePgpMimeSigned later
     const body: SendableMsgBody = { 'text/plain': newMsg.plaintext, 'text/html': newMsg.plainhtml };
-    const sendable = await SendableMsg.create(this.acctEmail, { ...this.headers(newMsg), body, atts, type: 'pgpMimeSigned' });
+    //const sendable = await SendableMsg.create(this.acctEmail, { ...this.headers(newMsg), body, atts, type: 'pgpMimeSigned' });
+    const sendable = await SendableMsg.createOpenPGPSigned(this.acctEmail, { ...this.headers(newMsg), body, atts });
```

Comment:
Here it's indeed fair to use a special `createOpenPGPSigned` but let's actually call it `createPgpMimeSigned` to be more precise.

Reply:
no problem